checks/remotesettings/certificates_expiration.py

```diff
@@ -0,0 +1,86 @@
+"""
+Signature certificates should expire in a minimum number of days.
+
+The list of collections whose certificate expires too soon is returned, along
+with the expiration date and x5u URL.
```

Comment:
I think this wording is a bit awkward. How about something like:

```suggestion
Signature certificates should not expire for at least some minimum number of days.

Returns a list of collections whose certificate expires too soon, along with their expiration date and x5u URL.
```